juego.py

```diff
@@ -146,6 +147,13 @@ class Game():
 
     def __init__(self, activity):
         self.activity = activity
+        self.locale = locale.getdefaultlocale()[0]
+        if self.locale not in ("en_US", "es_ES"):
+            self.locale = "en_US"
+        self.help_text = {
+            "en_US": "Select correct ball to answer or type it using keyboard",
+            "es_US": "Selecciona la bola correcta para responder o escribe la respuesta usando el teclado",
```

Comment:
Both values are `en_US` and that's not what you want.

This can also be done right outside the activity class in a function that returns the necessary text.